DEVELOPERS.md

```diff
@@ -9,29 +9,32 @@
       - [Installing on macOS](#installing-on-macos)
       - [Installing on Linux](#installing-on-linux)
       - [Creating a database](#creating-a-database)
-      - [Restoring Backups](#restoring-backups)
+    - [Restoring Backups](#restoring-backups)
     - [Steps](#steps)
   - [Docker Compose](#docker-compose)
   - [Frontend development (CSS/JS)](#frontend-development-cssjs)
   - [Running the local asset server](#running-the-local-asset-server)
   - [Compiling assets](#compiling-assets)
-  - [Data Setup](#data-setup)
-  - [Upgrade OpenTelemetry dependencies](#pip-opentelemetry)
+  - [Setting up a fresh install](#setting-up-a-fresh-install)
+  - [pip-opentelemetry](#pip-opentelemetry)
```

Comment:
thought: I'd have probably kept the "Upgrade OpenTelemetry dependencies" contents entry here and amended the section title, rather than the other way around, because "pip-opentelemetry" is not a very good title. But I don't have too strong opinions about it.

Reply:
Oh I thought my plugin just re-ordered them, didn't realise it changed the titles! Let me fix that.